### ISSUE TITLE
Fix transformers version problem

### ISSUE THREAD

**Issue by ahalterman** [ahalterman/NGEC-2025#3]
When refitting the Wiki XGboost model (#1), the accuracy is much lower than previously. I'm pretty sure this is because I switched the embedding model from Jina to Nomic. I did that because the Jina model had a problem when I loaded (some sort of torch method renaming).

In trying to fix that, I upgraded `torch`, `transformers`, and `sentence-transformers`, which hopelessly broke everything. 

First priority: get all of that working again. Then go back to #1 and finish that training.
